app/controllers/users/sessions_controller.rb

```diff
@@ -24,6 +24,7 @@ def new
 
       @ial = sp_session_ial
       @browser_is_ie11 = browser_is_ie11?
+      @is_on_home_page = is_on_home_page?
```

Comment:
Can we add some spec coverage for this instance variable? For example, there's currently a bug in how it's assigned since `|| "/{locale}"` will always result in it being `true`, which is unexpected.

Reply:
Actually, now that I think about it, do we need to do any logic at all, or could we assume that  the `SessionsController#new` action is definitely the homepage?

```suggestion
      @is_on_home_page = true
```